pipelines/datasets/br_bd_teste/tasks.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+"""
+Tasks for br_bd_teste
+"""
+
+###############################################################################
+#
+# Aqui é onde devem ser definidas as tasks para os flows do projeto.
+# Cada task representa um passo da pipeline. Não é estritamente necessário
+# tratar todas as exceções que podem ocorrer durante a execução de uma task,
+# mas é recomendável, ainda que não vá implicar em  uma quebra no sistema.
+# Mais informações sobre tasks podem ser encontradas na documentação do
+# Prefect: https://docs.prefect.io/core/concepts/tasks.html
+#
+# De modo a manter consistência na codebase, todo o código escrito passará
+# pelo pylint. Todos os warnings e erros devem ser corrigidos.
+#
+# As tasks devem ser definidas como funções comuns ao Python, com o decorador
+# @task acima. É recomendado inserir type hints para as variáveis.
+#
+# Um exemplo de task é o seguinte:
+#
+# -----------------------------------------------------------------------------
+# from prefect import task
+#
+# @task
+# def my_task(param1: str, param2: int) -> str:
+#     """
+#     My task description.
+#     """
+#     return f'{param1} {param2}'
+# -----------------------------------------------------------------------------
+#
+# Você também pode usar pacotes Python arbitrários, como numpy, pandas, etc.
+#
+# -----------------------------------------------------------------------------
+# from prefect import task
+# import numpy as np
+#
+# @task
+# def my_task(a: np.ndarray, b: np.ndarray) -> str:
+#     """
+#     My task description.
+#     """
+#     return np.add(a, b)
+# -----------------------------------------------------------------------------
+#
+# Abaixo segue um código para exemplificação, que pode ser removido.
+#
+###############################################################################
+
+from prefect import task
+
+
+@task  # noqa
+def say_hello(name: str = "World") -> str:
```

Comment:
Essa task existe apenas para dar um "início" ao desenvolvimento e testar configurações iniciais. Ela também deve ser eliminada.